src/instructlab/cli/data/generate.py

```diff
@@ -143,8 +143,8 @@
 @click.option(
     "--batch-size",
     type=click.IntRange(min=0),
-    default=None,
-    help="Number of elements to process in each batch through the SDG pipeline. Enabled by default for the vLLM serving backend, with a batch size of 8 chosen based on experiments to optimize for throughput. Use 0 to disable.",
+    cls=clickext.ConfigOption,
+    default=DEFAULTS.BATCH_SIZE,
```

Comment:
the default comes from the config when you set `cls=...` so no default is necessary here.

Reply:
Thanks @cdoern, hmm I think I am missing something. If I run `ilab data generate --debug-params`. I get all the values of the parameter correct except for the batch size for which I get 
```batch_size: None     	[type: None, src: default]``` (if I remove the default)